test/object-store/audit.cpp

```diff
@@ -1723,6 +1723,9 @@ TEST_CASE("audit integration tests") {
                                               REQUIRE_FALSE(error);
                                               deleted = *count;
                                           });
+                    if (deleted == 0) {
+                        millisleep(100); // don't spam the server too much
```

Comment:
what are these about? was there evidence that we were overloading the server?

Reply:
It was mainly done to reduce the log spamming since it repeated the loop a bunch of times before it completed

Reply:
Updated comment